api/data_refinery_api/test/test_api_general.py

```diff
@@ -306,6 +306,9 @@ def test_all_endpoints(self):
         response = self.client.get(reverse("create_dataset", kwargs={"version": API_VERSION}))
         self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
+        response = self.client.get(reverse("samples", kwargs={"version": API_VERSION}) + "?foo=bar")
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
Could you add an assertion to make sure `foo` made it into response's `invalid_filters`?